twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -121,6 +124,20 @@ def twitter_scrooge(
         actual = util_logging,
     )
 
+    mustache_name = "io_bazel_rules_scala_mustache"
```

Comment:
I'm also not seeing where this is used (since I didn't understand the reference in `src/scala/io/bazel/rules_scala/scrooge_support/BUILD`

Reply:
This is used by a [ApacheJavaGenerator.scala](https://github.com/twitter/scrooge/blob/develop/scrooge-generator/src/main/scala/com/twitter/scrooge/java_generator/ApacheJavaGenerator.scala) in Scrooge, to generate java from mustache templates. Since our scrooge wrapper inserts itself in scrooge in a funny way, it needs these dependencies declared for it.

Reply:
Also, mustache hasn't changed in a very long time, so I didn't think it needed to be customizable like the other targets here, but happy to change that.

Reply:
ok, I now think I know what's wrong with the current setup.
I think that the generator deps should be specified to it.
This [call](https://github.com/bazelbuild/rules_scala/blob/c346443ab84f4f60b6178bdd2899102951d3519b/twitter_scrooge/twitter_scrooge.bzl#L107) should pass in the runtime_deps (`scala_maven_import_external` takes in `deps` and `runtime_deps` attributes). This will clarify to the reader that our code doesn't need it but it's a requirement of the generator (and I'd maybe say it's a requirement to generate java code)

Reply:
Wow, that makes a lot of sense, and actually helps me understand another problem!
Made the change, thanks a lot for the help.